### PR TITLE
Add contact form

### PR DESCRIPTION
**Pull request by ameerhamzakhalid**
I added the contact form module in HTML.
I added the image required for the contact form section layout according to the template.
I added the CSS properties required for the contact form section module.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
...
+      <ul>
+        <li>
+          <input type="text" name="Name" placeholder="Full name">
+        </li>
+        <li>
+          <input type="email" name="Email" placeholder="Email address">
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
```
Comment (moise10r) [inline index.html]: - [ ] Please, according to the requirement, All input fields should have the `required` attribute to activate the validation on submit, which means, the user can only submit the form after filling all input fields, kindly make sure to add this.

File: index.html [hunk 2]
```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
...
+    <li>
+      <a href="">
+        <img src="./img/tweet.svg" alt="twitter icon">
+      </a>
+    </li>
+    <li>
+      <a href="">
+         <img src="./img/lnkdn.svg" alt="linkedin icon">
+      </a>
+    </li>
```
Comment (moise10r) [inline index.html]: - [x] Kindly use the `flexbox`  CSS property to layout the elements on this section.

File: index.html (hunk 1 above)
Comment (moise10r) [inline index.html]: - [ ] According to the project requirement: 

- Name field must accept up to 30 chars, which means, the Name input should have the `maxlength="30"` attribute.
- Message field accepts up to 500 chars which means, the Message input should have the `maxlength="500"` attribute.

File: index.html [hunk 3]
```diff
@@ -271,5 +271,59 @@ <h2 class="categorie-title">
...
+        </li>
+        <li>
+          <textarea name="Message" id="msg" cols="30" rows="10">Enter text here</textarea>
+        </li>
+        <li class="submit-button">
+          <button type="submit">
+              Get in touch
+          </button><br>
+          <small></small>
+      </li>
```
Comment (moise10r) [inline index.html:293]: - [ ] Button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template, Kindly add the `hover` effect to your `Get in touch` button look :arrow_down: 

```
button:hover {
   box-shadow: 0px 8px 16px rgba(54, 179, 127, 0.24);
}
```

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi @TEAM,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

### To highlight 💪🏻

- No linter errors. ✔️ 
- The Form uses the `action` and `method` attribute to integrate with [formspree service](https://formspree.io/html). ✔️ 
- The form is submitted successfully with valid input values.  ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [ ] Please, Write a meaningful title for your pull request. you can check the  [Professional repo - basic rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (ameerhamzakhalid) [reply index.html]: @moise10r Thank you very much for your kind response but I have already added the `flex box` CSS property in the `social-media` class. Isn't that okay?

Comment (moise10r) [reply index.html]: Yeah but It will be better if you can also add it to the social-media class or `ul` container

Comment (ameerhamzakhalid) [reply index.html]: @moise10r yeah I had added to the `social-media` class. But now I have added a new class `social-media-footer` and added `flex box` in it too.

Comment (moise10r) [reply index.html]: Great :+1:

**Review by elmar8287 — APPROVED** [review]
Hi, 

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**